packages/fuels-core/src/encoding_utils.rs

```diff
@@ -0,0 +1,105 @@
+use crate::{ParamType, WORD_SIZE};
+
+/// If given a non-empty slice it will return the number of `WORD`s needed to fit the biggest of the provided types.
+pub fn max_by_encoding_width(params: &[ParamType]) -> Option<usize> {
+    params.iter().map(encoding_width).max()
+}
+
+/// Calculates the number of `WORD`s the VM expects this parameter to be encoded in.
+pub fn encoding_width(param: &ParamType) -> usize {
```

Comment:
I'm not sure about the `get` part, seems like it can be added anywhere you're returning something.

But the expected part is great, communicates that we're only serving the VM.

Reply:
It's now called compute_encoding_width as per the other comment.